closure/templates/test/ExampleModule.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 The Closure Rules Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package test;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.inject.AbstractModule;
+import com.google.inject.multibindings.Multibinder;
+import com.google.template.soy.data.SoyValue;
+import com.google.template.soy.data.restricted.StringData;
+import com.google.template.soy.exprtree.Operator;
+import com.google.template.soy.jssrc.restricted.JsExpr;
+import com.google.template.soy.jssrc.restricted.SoyJsSrcFunction;
+import com.google.template.soy.jssrc.restricted.SoyJsSrcPrintDirective;
+import com.google.template.soy.shared.restricted.SoyFunction;
+import com.google.template.soy.shared.restricted.SoyJavaFunction;
+import com.google.template.soy.shared.restricted.SoyJavaPrintDirective;
+import com.google.template.soy.shared.restricted.SoyPrintDirective;
+import java.util.List;
+import java.util.Set;
+
+public class ExampleModule extends AbstractModule {
```

Comment:
This could use a comment pointing at https://github.com/google/closure-templates/blob/master/documentation/dev/plugins.md

Also, maybe remove the `SoyJavaFunction`, `rules_closure` doesn't support plugins for Java atm.

Reply:
Done